### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by YoussB**
rendered: https://github.com/concourse/resource-types-website/blob/update-readme/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,43 @@
...
+## Architecture:
 
-- Duty Free is hosted on github pages https://concourse.github.io/dutyfree
-- To generate the Duty Free html files locally, the main.go file is executed, passing a destination folder for the html files and [`resources.yml`](https://github.com/concourse/dutyfree/blob/master/resources.yml) as arguments, e.g. `go run main.go /tmp/folder resources.yml`
-- a Concourse pipeline for running tests and deploying to Github pages is available in the [`ci` folder](https://github.com/concourse/dutyfree/tree/master/ci); it requires 
-  - ssh key (as `ci.private`) for pushing to the `gh-pages` branch
-  - Github token (as `github_token`) to retrieve the ReadMe of the resources from Github 
-  - Slack hook, to notify about a new push deployment of the website
-- The website is rebuilt every day (9am UTC) to refresh the content of ReadMe of the resources
+- The code is mainly a Golang backend with an elm fronent.
```
Comment (jomsie) [inline README.md]: front-end or frontend

File: README.md [hunk 2]
```diff
@@ -1,22 +1,43 @@
-# DutyFree
-Duty Free is a website for showcasing re-usable components of Concourse. With DutyFree, Concourse users can explore and discover resources written by the community 
+# Resource Types Website for Concourse
+Welcome to the development repository for the resource types website. Here you can find the source code for the website as well as different options for packaging. The website is currently running under the Concourse docs website (resource types tab) or as a [standalone website](https://resource-types.concocurse-ci.org)
```
Comment (jomsie) [inline README.md]: its linked from the docs website, and it is a standalone website?

File: README.md [hunk 3]
```diff
@@ -1,22 +1,43 @@
...
+## Run Locally:
+There are multiple ways to run the code locally:
 
-`go run dev/dev.go`
+- to run directly from the code, you can run the following commands:
+  - if you want to use a different port, the app uses the environment variable `PORT` to select its port or else it will use port `9090`
+  ```bash
+  cd web
+  yarn build && yarn install
+  cd ../warhouse
```
Comment (jomsie) [inline README.md]: warehouse

File: README.md [hunk 4]
```diff
@@ -1,22 +1,43 @@
...
-## Run a development server
+## Run Locally:
+There are multiple ways to run the code locally:
 
-`go run dev/dev.go`
+- to run directly from the code, you can run the following commands:
+  - if you want to use a different port, the app uses the environment variable `PORT` to select its port or else it will use port `9090`
+  ```bash
+  cd web
+  yarn build && yarn install
```
Comment (jomsie) [inline README.md]: install then build

File: README.md [hunk 5]
```diff
@@ -1,22 +1,43 @@
...
+- To test the backend server:
+  ```bash
+  cd warehouse
+
+  go get -u github.com/onsi/ginkgo/ginkgo
+
+  ginkgo -r -keepGoing .
+  ```
+
+- To test the fronend code: `yarn install && yarn build`
```
Comment (jomsie) [inline README.md]: yarn test

**Comment by YoussB** [comment]
@jomsie Done

**Pull request merged**